nms/app/packages/magmalte/app/views/equipment/EnodebDetailConfigEdit.js

```diff
@@ -519,6 +519,17 @@ export function ConfigEdit(props: Props) {
           return;
         }
       }
+
+      // fill in default or undefined values here
```

Comment:
Please add a `TODO` with github user ID attached to remove these two blocks later.

Reply:
This is not a todo, this is actually describing what i'm doing here